controllers/operator/objects.go

```diff
@@ -3,6 +3,8 @@ package operator
 import (
 	"fmt"
 
+	"os"
+
```

Comment:
```suggestion
	"fmt"
	"os"

```

in general we are keeping this order:
1. golang packages
2. k8s
3. sig.k8s
4. openshift
5. openshift/multiarch-tuning-operator

Reply:
or try to :). The linter does only a part of it